frontend/pages/hosts/HostDetailsPage/SoftwareListRow/SoftwareListRow.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+
+import softwareInterface from "interfaces/software";
+
+const baseClass = "software-list-row";
+
+class SoftwareListRow extends Component {
+  static propTypes = {
+    software: softwareInterface.isRequired,
+  };
+
+  render() {
+    const { software } = this.props;
+    const { name, source, version } = software;
+
+    const TYPE_CONVERSION = {
+      apt_sources: "Package (APT)",
+      deb_packages: "Package (deb)",
+      portage_packages: "Package (Portage)",
+      rpm_packages: "Package (RPM)",
+      yum_sources: "Package (YUM)",
+      npm_packages: "Package (NPM)",
+      atom_packages: "Package (Atom)",
+      python_packages: "Package (Python)",
+      apps: "Application (macOS)",
+      chrome_extensions: "Browser plugin (Chrome)",
+      firefox_addons: "Browser plugin (Firefox)",
+      safari_extensions: "Browser plugin (Safari)",
+      homebrew_packages: "Package (Homebrew)",
+      programs: "Program (Windows)",
+      ie_extensions: "Browser plugin (IE)",
+      chocolatey_packages: "Package (Chocolatey)",
+      pkg_packages: "Package (pkg)",
+    };
+
+    const type = TYPE_CONVERSION[source];
```

Comment:
```suggestion
    const type = TYPE_CONVERSION[source] || "Uknown";
```
I'm confident that the `source` property for each software item will always map to a value in the `TYPE_CONVERSION` dictionary. Just in case we hit the case where the `source` doesn't exist, maybe we should default to something like "Unkown."